app/templates/home.html

```diff
@@ -7,6 +7,8 @@
             Torchvision model zoo. Go
         <a href={{url_for("create_classify")}}>here</a> for
         trying out the service.</p>
+    <p style="margin-top:1cm;">Click
+        <a href={{url_for("create_histogram")}}>here</a> if you want to compute the histogram of a chosen image.</p>
```

Comment:
Suggestion (it would be great if you fix it but it is not mandatory): to improve the usability, shouldn't be better to put this link in the navigation bar (moving it to `base.html`)?